TestAssets/DesktopTestProjects/AppWithProjTool2Fx/App.csproj

```diff
@@ -6,7 +6,7 @@
     <OutputType>Exe</OutputType>
     <AssetTargetFallback Condition=" '$(TargetFramework)' == 'netcoreapp3.0' ">$(AssetTargetFallback);portable-net45+win8;dnxcore50</AssetTargetFallback>
     <RuntimeIdentifiers>win7-x64;win7-x86;osx.10.10-x64;osx.10.11-x64;ubuntu.14.04-x64;ubuntu.16.04-x64;ubuntu.16.10-x64;centos.7-x64;rhel.7.2-x64;debian.8-x64;fedora.24-x64;opensuse.42.1-x64</RuntimeIdentifiers>
-    <RestoreAdditionalProjectSources Condition="'$(TEST_PACKAGES)' != ''">$(TEST_PACKAGES)</RestoreAdditionalProjectSources>
+    <RestoreAdditionalProjectSources Condition="'$(TEST_PACKAGES)' != ''">$(RestoreAdditionalProjectSources);$(TEST_PACKAGES)</RestoreAdditionalProjectSources>
```

Comment:
Why did this only become an issue now?

Reply:
I think before we were just getting lucky that the projects were being built in the right order.

Reply:
@livarcocc I think during moving to arcade we make all test package to be in one place that is the "default"